src/jwtauthmiddleware.js

```diff
@@ -35,6 +42,11 @@ function jwtAuthMiddleware(pubKeys, audiences, mapper = null, options = {}) {
       if (!decodedJwtBody.sub) {
         return next(new JwtVerifyError(`Missing 'sub' in body`))
       }
+      for (const token of revokedTokens.tokens) {
+        if (token.jti == decodedJwtBody.jti) {
+          return next(new JwtVerifyError(`Revoked token`))
+        }
+      }
```

Comment:
Looking at this I wonder whether we're opening ourselves up to an "easy" way of DoS'ing or slowing down all our requests by creating a script that just creates and revokes tokens as fast as possible. Perhaps we should pass this in as an object instead so we can just do a constant lookup. Then the old issue with a lot of tokens is memory consumption and that would need to be an insane amount of revoked tokens to be an issue.

That would also allow us to not need to nest the array as we can just pass in the map object directly with this structure:
```
const revokedTokens: Record<string, Date> = { jti: new Date('2022-08-06T13:37:00Z') }
```

And then check for revocation by during:
```
if (revokedTokens[decodedJwtBody.jti]) {
  return next(new JwtVerifyError('Revoked token'))
}
```